mantidproject/mantid
# Updates to Basic Course

**Description of work.**

General Aim:  Updating the Mantid Online Courses for Workbench 

Little Bits I've done:

- Corrected toctree to fix the structure of the Basic Courses sections
- Altered MantidPlot wording.
- Changed images to be of workbench
- Included a short MantidPlot section
- Included a useful section
- Added in an end page 

**To test:**



- Check images are appropriate sizes and pages are generally well formatted
- Check the page ordering is correct and all links on each index page work.
- Check as many exercises as you can (get others to help if this is taking too long)
- Check the links in 03_workspaces.html in the algorithms section work
- Need to check that the generate script bit works in Part 3 of 07_exercises.html.
- Workspaces have been explained in many places (check this makes sense) I thought it was better to introduce the idea earlier. 
- Any suggestions for making the course less boring would be great!
(- I would like to add a jump back up to the contents page button in the bottom navigation bar.)
refs #25796

*This does not require release notes* because **documentation change**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
